modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -28,17 +30,17 @@ The following `PtpConfig` CR configures PTP fast events by setting values for `p
 apiVersion: ptp.openshift.io/v1
 kind: PtpConfig
 metadata:
-  name: boundary-clock-ptp-config <1>
+  name: boundary-clock-ptp-config                  <1>
   namespace: openshift-ptp
 spec:
-  profile: <2>
-  - name: "profile1" <3>
-    interface: "" <4>
-    ptp4lOpts: "-2 -s --summary_interval -4" <5>
-    ptp4lConf: | <6>
-      [ens1f0] <7>
+  profile:                                         <2>
+  - name: "<profile_name>"                         <3>
+    interface: ""                                  <4>
+    ptp4lOpts: "-2 -s --summary_interval -4"       <5>
```

Comment:
Since this is the BC profile, the **-s** option should not be added. Command line options have preferences over ptp4l.conf profile values and one of the port must be selected as MASTER port (ens1f3 in this example which has masterOnly option set to one).